pkg/releasetesting/environment.go

```diff
@@ -31,10 +32,13 @@ import (
 
 // Environment encapsulates information about where test suite executes and returns results
 type Environment struct {
-	Namespace  string
-	KubeClient environment.KubeClient
-	Stream     services.ReleaseService_RunReleaseTestServer
-	Timeout    int64
+	Namespace   string
+	KubeClient  environment.KubeClient
+	Stream      services.ReleaseService_RunReleaseTestServer
+	Timeout     int64
+	Parallel    bool
+	Concurrency uint32
```

Comment:
Should we change the word `Concurrency` here to `Parallelism`? This can get a little confusing because parallelism and concurrency are related but slightly different. CircleCI has a similar concept for running tests faster and they use a key called `Parallelism` to specify how many independent executors you'd want to set

Reply:
I don't have a strong preference either way, i went with concurrency because that's what I've seen more often in other software (e.g. `ab`)

Reply:
sure, done and re-pushed 🎉